proc_square_refl.py

```diff
@@ -143,19 +143,28 @@ def filter_range(x): return array([j for j in x if
     d.setaxis('t', lambda x: x-time_zero).register_axis({'t':0})
     refl_blip_ranges -= time_zero
     pulse_range -= time_zero
-    fl.next('after setting t=0')
-    for j in range(2):
-        fl.plot(abs(d['ch',j]), linewidth=3, color='k',
-                label='ch %d abs'%(j+1), alpha=0.3)
     d = d['t':(-10e6,10e6)] # slice out frequencies with signal
     #}}}
     
     #{{{zeroth order phase correction
     for j in range(2):
+        fl.basename = "channel %d"%(j+1)
         ph0 = zeroth_order_ph(d['ch',j], fl=fl)
         d['ch',j] /= ph0
+    fl.basename = None
     #}}}
 
+    # {{{ 
+    fl.next('after all corrections are complete')
+    for j in range(2):
+        fl.plot(d['ch',j].real,
+                label='ch %d real'%(j+1), alpha=0.5)
+        fl.plot(d['ch',j].imag,
+                label='ch %d imag'%(j+1), alpha=0.5)
+        fl.plot(abs(d['ch',j]), linewidth=3, color='k',
+                label='ch %d abs'%(j+1), alpha=0.3)
+    # }}}
```

Comment:
this shows what the data looks like after all corrections have been applied, and it's not pretty.  There is a very significant imaginary component (I'm a little worried, b/c this should have been caught earlier).  Based on the diagnostic plots, I do think this will be fixed by solving #23 , so we can close the pull request w/out solving this right now, but you do need to start a new branch to address #23 as soon as you can.